rabbitmq/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export HOME={{pkg.svc_var_path}}
+export RABBITMQ_MNESIA_BASE={{pkg.svc_var_path}}/db
+export RABBITMQ_LOG_BASE={{pkg.svc_var_path}}/log
+export RABBITMQ_CONFIG_FILE={{pkg.svc_config_path}}/rabbitmq
+
+exec {{pkg.path}}/sbin/rabbitmq-server 2>&1
```

Comment:
this plan doesn't restart RabbitMQ correctly, try something like this:

```
hab config apply rabbitmq.default 1 ./foo.toml --peer 172.17.0.2:9634

...
Writing new file from gossip: /hab/svc/rabbitmq/gossip.toml
rabbitmq(SV): Stopping
hab-sup(SV): rabbitmq - process 300 died with exit code 143
hab-sup(SV): rabbitmq - Service exited
rabbitmq(SV): Starting
rabbitmq(O): ERROR: node with name "rabbit" already running on "4b8309a8d6f0"
hab-sup(SV): rabbitmq - process 555 died with exit code 1
hab-sup(SV): rabbitmq - Service exited
rabbitmq(SV): Starting
```

with `foo.toml` containing arbitrary toml to trigger a restart.